package.json

```diff
@@ -38,11 +38,11 @@
     "gulp": "^4.0.2",
     "gulp-append-prepend": "^1.0.8",
     "gulp-csso": "^4.0.1",
+    "gulp-dart-sass": "^1.0.2",
     "gulp-if": "^3.0.0",
     "gulp-inject-string": "^1.1.2",
     "gulp-postcss": "^9.0.0",
     "gulp-rename": "^2.0.0",
-    "gulp-sass": "^4.1.0",
```

Comment:
@tywrenn , Is the transfer from gulp-sass to gulp-dart-sass testing well. If so, I may separate this out as a quick separate PR to get into the codebase so can then do a buildx with the arm64 for the openemr:6.0.0 docker.